### PR TITLE
[11.0][MIG] account reversal: Migration to 11.0

### PR DESCRIPTION
**Pull request by pedrobaeza**
Migration commit is d67419f 

@Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
@luismontalba @cubells please review

File: account_reversal/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+* Allowing inheritance
+* Options like prefix (for journal entry and journal item), post and reconcile.
+* Create a link between the entry and its reversal
+* Mark entries to be reversed in the future.
+
+Usage
+=====
+
+As in Odoo v11c, if you select an entry from Accounting > Adviser > Journal Entries,
```
Comment (cubells) [inline account_reversal/README.rst]: The exact menu is `Accounting > Adviser > Accounting entries > Journal Entries`

File: account_reversal/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "author": "Akretion,"
+              "Camptocamp,"
+              "ACSONE SA/NV,"
+              "Tecnativa,"
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "account",
```
Comment (cubells) [inline account_reversal/__manifest__.py]: To be able to do tests, I had to install account_invoicing.

File: account_reversal/i18n/account_reversal.pot [hunk 3]
```diff
@@ -0,0 +1,132 @@
+# Translation of OpenERP Server.
```
Comment (cubells) [inline account_reversal/i18n/account_reversal.pot]: Don't you have to remove this file?

**Review by cubells — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
@cubells thanks for the review. Changes done

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

code review and test.

**Comment by pedrobaeza** [comment]
@luismontalba please review

**Comment by luismontalba** [comment]
Can't access runbot, neither admin/admin nor demo/demo

**Comment by pedrobaeza** [comment]
Runbot seems to be having issues. Please try tomorrow afternoon rebuilding.

**Comment by luismontalba** [comment]
Some issue than yesterday ...

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Comment by pedrobaeza** [comment]
Closing as a later PR has been merged.

**Pull request closed without merging**